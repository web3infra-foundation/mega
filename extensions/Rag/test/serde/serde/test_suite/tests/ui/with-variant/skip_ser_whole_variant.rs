use serde_derive::Serialize;

#[derive(Serialize)]
enum Enum {
    #[serde(serialize_with = "serialize_some_unit_variant")]
    #[serde(skip_serializing)]
    Unit,
}

fn main() {}
