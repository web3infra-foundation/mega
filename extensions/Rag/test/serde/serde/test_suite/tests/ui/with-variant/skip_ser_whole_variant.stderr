error: variant `Unit` cannot have both #[serde(serialize_with)] and #[serde(skip_serializing)]
 --> tests/ui/with-variant/skip_ser_whole_variant.rs:5:5
  |
5 | /     #[serde(serialize_with = "serialize_some_unit_variant")]
6 | |     #[serde(skip_serializing)]
7 | |     Unit,
  | |________^
